src/components/history-page/history-content/history-content.tsx

```diff
@@ -58,7 +65,29 @@ export const HistoryContent: React.FC<HistoryContentProps> = ({ viewState, entri
     removeHistoryEntry(noteId).catch(showErrorNotification('landing.history.error.deleteEntry.text'))
   }, [])
 
-  if (entries.length === 0) {
+  const historyContent = useMemo(() => {
+    return historyToolbarState.viewState === ViewStateEnum.TABLE ? (
```

Comment:
I'd argue that we want to keep the switch statement here…

Reply:
I don't really like the switch statement.. But I don't like the structure of the history page in general. I reverted the change here. We should refactor the history page.

Reply:
I guess the refactor will be necessary anyway when we exchange the history page with the explore page...